tests/dummy/app/routes/application.js

```diff
@@ -2,9 +2,9 @@ import Route from '@ember/routing/route';
 import { mockRequestsWorker } from 'dummy/mocks/worker';
 
 export default class ApplicationRoute extends Route {
-  // beforeModel() {
-  //   return mockRequestsWorker.start({
```

Comment:
What was the issue again? We should get that resolved so you don't have to keep commenting this locally 😄

Reply:
Haha I indeed commented this part everytime I wanted to edit something. Just ran the dummy app without commenting it and it did nothing strange! So you probably fixed it? Or mAgIc 😄